wpimath/src/main/java/edu/wpi/first/math/system/plant/DCMotor.java

```diff
@@ -71,15 +71,15 @@ public double getTorque(double currentAmpere) {
    * Calculate the voltage provided to the motor at a given torque and angular velocity.
    *
    * @param torqueNm The torque produced by the motor.
-   * @param speedRadiansPerSec The speed of the motor.
+   * @param speedRadiansPerSec The current angular velocity of the motor.
    * @return The voltage of the motor.
    */
   public double getVoltage(double torqueNm, double speedRadiansPerSec) {
     return 1.0 / KvRadPerSecPerVolt * speedRadiansPerSec + 1.0 / KtNMPerAmp * rOhms * torqueNm;
   }
 
   /**
-   * Calculate the speed of the motor at a given torque and input voltage.
+   * Returns the angular speed produced by the motor at a given torque and input voltage.
```

Comment:
Make the other descriptions match the C++ version too while you're at it.

Reply:
they should be on parity now, unless you wanted them to match exactly